### PR TITLE
feature(agent): add support for Universal eBPF

### PR DESCRIPTION
**Pull request by mavimo**
## What this PR does / why we need it:

 - Add support for the Universal eBPF driver

## Checklist

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [x] All test files are added in the tests folder of their respective chart and have a "_test" suffix

### CODE REVIEW

File: charts/agent/templates/daemonset.yaml [hunk 1]
```diff
@@ -95,6 +96,13 @@ spec:
             - name: SYSDIG_BPF_PROBE
               value:
           {{- end }}
+          {{- if (include "agent.universalEbpfEnforced" .) }}
```
Comment (AlbertoBarba) [inline charts/agent/templates/daemonset.yaml]: I think this env is not needed...if `agent.universalEbpfEnforced` returns true we don't start the `initContainer`

**Review by AlbertoBarba — CHANGES_REQUESTED** [review]

**Review by AlbertoBarba — APPROVED** [review]

File: charts/agent/tests/universal_ebpf_test.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+suite: Universal eBPF tests
+templates:
+  - templates/daemonset.yaml
+tests:
+  - it: Ensure that when the eBPF is disabled we create the sysdig container wihtout SYSDIG_BPF_PROBE and SYSDIG_AGENT_DRIVER environment variables
```
Comment (aroberts87) [inline charts/agent/tests/universal_ebpf_test.yaml]:
```suggestion
  - it: Ensure that when the eBPF is disabled we create the sysdig container without SYSDIG_BPF_PROBE and SYSDIG_AGENT_DRIVER environment variables
```

File: charts/agent/values.yaml [hunk 3]
```diff
@@ -169,6 +169,10 @@ priorityClassValue: 10
 ebpf:
   # Enable eBPF support for Sysdig Agent
   enabled: false
+
+  # Define the kind of eBPF driver that can be used in the agent. Can be `legacy_ebpf` or `universal_ebpf`
```
Comment (aroberts87) [inline charts/agent/values.yaml]:
```suggestion
  # Define the kind of eBPF driver that will be used by the agent. Can be `legacy_ebpf` or `universal_ebpf`
```

**Review by aroberts87 — APPROVED** [review]
LGTM! Just two small nits.

**Pull request merged**